en/mavgen_python/README.md

```diff
@@ -202,7 +207,7 @@ def system_time_send(self, time_unix_usec, time_boot_ms, force_mavlink1=False):
     '''
 ```
 
-If you're using **mavutil** for link management then the `mav` attribute provides access to a configured `MAVLink` class object that you can use for sending messages. 
+If you are using **mavutil** for link management then the `mav` attribute provides access to a configured `MAVLink` class object that you can use for sending messages.
```

Comment:
FYI I would not make this kind of change. The original is correct and the new one is in no way "better".

Reply:
Got it

Reply:
Guess I tried to be "technical" 😄